chrome-launcher/README.md

```diff
@@ -125,5 +125,9 @@ before_script:
   - export DISPLAY=:99.0
   - export CHROME_PATH="$(pwd)/chrome-linux/chrome"
   - sh -e /etc/init.d/xvfb start
-  - curl -L https://raw.githubusercontent.com/GoogleChrome/lighthouse/v2.1.0/lighthouse-core/scripts/download-chrome.sh | bash
+  - sleep 3 # wait for xvfb to boot
+addons:
+  chrome: stable
 ```
+
+> Since Travis can install Chrome at a run time you no longer need to install it manually. Read [more](https://docs.travis-ci.com/user/chrome).
```

Comment:
maybe this line makes more sense right below the `curl` command?

Reply:
actually, it's a little confusing to have the `curl` command above then not have it in the `travis.yml` snippet. What do  you think about rearranging to something like

```
In a CI environment like Travis, Chrome may not be installed. If you want to use `chrome-launcher`, Travis can [install Chrome at run time with an addon](https://docs.travis-ci.com/user/chrome).

> You can also install Chrome manually using Lighthouse's `download-chrome.sh` script:
> `curl -L https://raw.githubusercontent.com/GoogleChrome/lighthouse/v2.1.0/lighthouse-core/scripts/download-chrome.sh | bash`
```

then just the snippet changes you have currently:

````
Then in `.travis.yml`, use it like so:

```yaml
language: node_js
install:
  - yarn install
before_script:
  - export DISPLAY=:99.0
  - export CHROME_PATH="$(pwd)/chrome-linux/chrome"
  - sh -e /etc/init.d/xvfb start
  - sleep 3 # wait for xvfb to boot

addons:
  chrome: stable
```
````